airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/DebeziumPropertiesManager.java

```diff
@@ -89,10 +93,13 @@ public Properties getDebeziumProperties() {
     // By default "decimal.handing.mode=precise" which's caused returning this value as a binary.
     // The "double" type may cause a loss of precision, so set Debezium's config to store it as a String
     // explicitly in its Kafka messages for more details see:
-    // https://debezium.io/documentation/reference/2.1/connectors/postgresql.html#postgresql-decimal-types
+    // https://debezium.io/documentation/reference/2.2/connectors/postgresql.html#postgresql-decimal-types
     // https://debezium.io/documentation/faq/#how_to_retrieve_decimal_field_from_binary_representation
     props.setProperty("decimal.handling.mode", "string");
 
+    // https://debezium.io/documentation/reference/2.2/connectors/postgresql.html#postgresql-property-max-queue-size-in-bytes
+    props.setProperty("max.queue.size.in.bytes", BYTE_VALUE_256_MB);
```

Comment:
`max.queue.size.in.bytes` is only applicable to the Postgres Debezium connector. Is it ok to set this property in this class because it appears to be set for all CDC source connectors?

Reply:
Oh no its applicable for all the 3 connectors. I confirmed in the docs that's why decided to add it in the common properties section

Reply:
@prateekmukhedkar @subodh1810 fyi this indirectly exposes a downstream debezium bug re approximate queue size calculator.  see [issues-28069](https://github.com/airbytehq/airbyte/issues/28069#issuecomment-1750030783) for more info.  The downstream bug is fixed from debezium 2.4.0+.